Atomic/run.py

```diff
@@ -38,6 +39,17 @@ def run(self):
             args = self._get_args("RUN")
             if args:
                 args += self.command
+                opts_file = self._get_args("RUN_OPTS_FILE")
+                if opts_file:
+                    opts_file = self.sub_env_strings("".join(opts_file))
+                    if opts_file.startswith("/"):
+                        try:
+                            run_opts = open(opts_file, "r").read()
+                            os.environ["RUN_OPTS"] = run_opts
+                        except IOError:
+                            util.write_out("Failed to read RUN_OPTS_FILE %s" % opts_file)
```

Comment:
If you don't think this is an error then don't state anything.  If we want to report a problem it should be a write_err() anyways.